components/net_traits/lib.rs

```diff
@@ -449,16 +449,22 @@ pub struct ResourceFetchTiming {
     pub response_start: u64,
     pub fetch_start: u64,
     // pub response_end: u64,
-    // pub redirect_start: u64,
+    pub redirect_start: u64,
     // pub redirect_end: u64,
     // pub connect_start: u64,
     // pub connect_end: u64,
 }
 
+enum RedirectStartValue {
+    Zero,
```

Comment:
We will need to add `#[allow(dead_code)]` above the `Zero` variant to avoid warnings, since we don't actually use it yet.